### PR TITLE
Needs-CLA from First Time Contributors

### PR DESCRIPTION
**Pull request by Trenly**

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/109224#issuecomment-1582867342

cc @denelon

**Comment by Trenly** [comment]
Waiting on a fix from the GitOps team

**Review by ItzLevvie — APPROVED** [review]

File: .github/policies/labelManagement.issueOpened.yml [hunk 1]
```diff
@@ -15,6 +15,13 @@ configuration:
               action: Opened
         then:
           - addCodeFlowLink
+          # If the user is a first-time contributor, add the Needs-CLA Label
+          - if:
+              - activitySenderHasAssociation:
+                  association: FIRST_TIME_CONTRIBUTOR
```
Comment (stephengillie) [inline .github/policies/labelManagement.issueOpened.yml:21]: Is there an index or listing of the available terms?

Comment (Trenly) [reply .github/policies/labelManagement.issueOpened.yml:21]: Yes, and this is one of the allowed values. The GitOps team is working on a patch

Comment (Trenly) [reply .github/policies/labelManagement.issueOpened.yml:21]: Actually, after reviewing, the patch should be live. I'll pull in master and see if the issue persists

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**